passthrough_test.go

```diff
@@ -72,8 +71,8 @@ func TestNewPassthroughCmd(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-
-			cmd := NewPassthroughCmd(tt.versions)
+			
+			cmd, _ := NewPassthroughCmd(tt.versions)
```

Comment:
can add test case for this later, needs to be refactored slightly, but also the validate is being covered in a separate test.